controllers/dspipeline_params.go

```diff
@@ -630,6 +635,20 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 			}
 		}
 
+		if cfg := p.APIServer.CustomKfpLauncherConfig; cfg != "" {
+			if cm, err := util.GetConfigMap(ctx, cfg, p.Namespace, client); err != nil {
+				// If the custom kfp-launcher configmap is not available, that is OK
+				if !apierrs.IsNotFound(err) {
+					log.Error(err, fmt.Sprintf("Encountered error when attempting to fetch ConfigMap: [%s], Error: %v", cfg, err))
+					return err
+				}
```

Comment:
this should be 

```
if err != nil
```
we throw an error regardless of whether it's a notfound error or some other error